### DISCUSSION TITLE
Reset friend database?

### DISCUSSION THREAD

**Discussion by KodiacStorm** [vrcx-team/VRCX#712] [Q&A] [answered]
Hello I would like to reset all the information recorded on my friends etc (last login, time together, etc) for the new years. How do I do this? Thank youuu!

**Comment by Natsumi-sama** [comment, marked as answer]
Easiest way is to fully reset VRCX this can be done by deleting your database in `%AppData%\VRCX\VRCX.sqlite3` but if you want to only delete gameLogs (this is used for join count and time together) you can open the database with https://sqlitebrowser.org/ and delete your `gamelog_join_leave` table.

**Comment by KodiacStorm** [comment]
Thanks for quick answer!
